Tests/helper.py

```diff
@@ -15,6 +15,9 @@
 
 from PIL import Image, ImageMath, features
 
+# overridden in conftest.py
+pytestconfig = {"option": {"verbose": 0}}
```

Comment:
This was previously discussed in https://github.com/python-pillow/Pillow/pull/6653#discussion_r1060437395
![Screenshot 2023-05-08 at 7 30 46 am](https://user-images.githubusercontent.com/3112309/236703554-7a857cd1-a5bf-4049-8486-c3771b636703.png)